### PR TITLE
Update FluxC to 1.6.2

### PR DESCRIPTION
**Pull request by mchowning**
Updates the fluxc version in preparation for incorporating Mobile Gutenberg v1.22.0.

PR submission checklist:

- [X] I have considered adding unit tests where possible.
- [X] I have considered adding accessibility improvements for my changes.
- [X] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -96,7 +96,7 @@ buildScan {
 
 ext {
     daggerVersion = '2.22.1'
-    fluxCVersion = 'c5ea220f70c3a77681d376a17ce5c57e26c1c392'
+    fluxCVersion = '1.6.2'
```
Comment (hypest) [inline build.gradle]: Looks like the 1.6.2 tag is pointing to https://github.com/wordpress-mobile/WordPress-FluxC-Android/commit/a38f48b532d3f2e3605d5024bce04f1149e3c52c, a commit quite older than the hash it replaces (`c5ea220f70c3a77681d376a17ce5c57e26c1c392`).

Can you elaborate on this @mchowning ? Thanks!

Comment (mchowning) [reply build.gradle]: >Looks like the 1.6.2 tag is pointing to wordpress-mobile/WordPress-FluxC-Android@a38f48b, a commit quite older than the hash it replaces (c5ea220f70c3a77681d376a17ce5c57e26c1c392).

I believe [1.6.2/a38f48b](https://github.com/wordpress-mobile/WordPress-FluxC-Android/commit/a38f48b532d3f2e3605d5024bce04f1149e3c52c) is actually newer than the original commit (`c5ea220f70c3a77681d376a17ce5c57e26c1c392`). Let me know if I'm missing something.

https://github.com/wordpress-mobile/WordPress-FluxC-Android/commits/a38f48b532d3f2e3605d5024bce04f1149e3c52c
Screen Shot 2020-02-07 at 10 47 22 AM

Comment (mchowning) [reply build.gradle]: I believe [1.6.2/a38f48b](https://github.com/wordpress-mobile/WordPress-FluxC-Android/commit/a38f48b532d3f2e3605d5024bce04f1149e3c52c) is actually newer than the original commit (`c5ea220f70c3a77681d376a17ce5c57e26c1c392`). Let me know if I'm missing something.

https://github.com/wordpress-mobile/WordPress-FluxC-Android/commits/a38f48b532d3f2e3605d5024bce04f1149e3c52c
Screen Shot 2020-02-07 at 10 47 22 AM

**Comment by mchowning** [comment]
:wave: @hypest ! Based on our offline discussion, I have created a 1.6.1.1 FluxC release that matches exactly to the hash of fluxc that we were using before.

File: build.gradle (hunk 1 above)
Comment (hypest) [reply build.gradle]: Ah, you are right! Thanks for double checking!

**Pull request merged**